src/frontend/apps/standalone_site/src/features/Playlist/hooks/useSelectPlaylist.tsx

```diff
@@ -33,12 +33,18 @@ const messages = defineMessages({
 interface UseSelectPlaylistOptions {
   formFieldProps?: Partial<FormFieldExtendedProps>;
   selectProps?: Partial<SelectExtendedProps>;
+  withPlaylistCreation?: boolean;
 }
 
-const useSelectPlaylist = ({
-  formFieldProps,
-  selectProps,
-}: UseSelectPlaylistOptions = {}) => {
+const useSelectPlaylist = (
+  {
+    formFieldProps,
+    selectProps,
+    withPlaylistCreation,
+  }: UseSelectPlaylistOptions = {
+    withPlaylistCreation: false,
+  },
```

Comment:
I don't think that you need to set it, if it's undefined it will act like false.

```suggestion
    withPlaylistCreation,
  }: UseSelectPlaylistOptions,
```